packages/vue_frontend_rpc/src/components/Editor.vue

```diff
@@ -130,3 +119,67 @@ export default {
   },
 };
 </script>
+
+<style>
+html,
+body {
+  height: 100%;
+  padding: 0px;
+}
+
+.inquirer-gui {
```

Comment:
Did you have to adjust so many styles from the inquirer-gui? shouldn't they be consumed as is with their styles?

Reply:
i removed this style